### ISSUE TITLE
[Accessibility] Narrator cannot announce the progress of the VScrollbar/HScrollbar when updating it

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#9158]
### .NET version

.Net 8.0 SDK build: 8.0.100-preview.5.23268.13

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No

### Issue description

When using keyboard to update the progress of the VScrollbar/HScrollbar controls, Narrator cannot announce the progress updating

.Net 8.0:
![image](https://github.com/dotnet/winforms/assets/26474449/e472cdb7-487e-4ea2-85b7-90487058670b)
![ScrollbarProgress](https://github.com/dotnet/winforms/assets/26474449/915e9726-23d9-4bba-a659-f735a552467e)
![image](https://github.com/dotnet/winforms/assets/26474449/bf38e7d2-95eb-471c-883a-e88bca62831c)

### Steps to reproduce

Repro steps:
1. Create a Winforms .net core application with VScrollbar/HScrollbar
2. Build and run application
3. Launch Narrator, and use keyboard to focus on VScrollbar/HScrollbar control to update its progress 

or use this sample application:
[WinFormsApp19.zip](https://github.com/dotnet/winforms/files/11529647/WinFormsApp19.zip)

**Comment by Tanya-Solyanik** [comment]
@LeafShi1  - this might be a good first issue, please work with @dmitrii-drobotov on it.

**Comment by Ashley-Li** [comment]
Verified this issue in the latest .NET 8.0 SDK build: 8.0.100-preview.6.23315.27, it was fixed. Now when using keyboard/mouse to update the progress of the VScrollbar/HScrollbar controls, Narrator can announce the progress updating. Also, would it be better to add % to the progress value ? as it shows 0% when start reading.
![9158_issue](https://github.com/dotnet/winforms/assets/56664234/af605e21-b12d-4e74-bee1-3ffe01a0cabb)

**Comment by Tanya-Solyanik** [comment]
@Ashley-Li - do Office apps or WPF apps announce `%` when scrolling?

**Comment by Ashley-Li** [comment]
@Tanya-Solyanik Tested Office apps(Outlook/OneNote) and WPF applications(set ScrollBar control's IsTabStop=true), Narrator cannot focus on the scrollbar and cannot announce.

**Comment by Tanya-Solyanik** [comment]
Thank you for the experiment, @Ashley-Li. I think that our fix makes sense because narrator wouldn't have type to say word `percent` after each number, and when it doesn't have time to read the complete announcement, it starts  `throttling`, skipping some announcements. Additionally, the user would not want to hear `percent` repeated all the time. I think the fix is sufficient as is.

**Comment by Philip-Wang01** [comment]
Verified with .NET SDK 8.0.100-preview.6.23327.6 test pass build, this issue is fixed. Now when using keyboard/mouse to update the progress of the VScrollbar/HScrollbar controls, Narrator can announce the progress updating.
![testscrollbar](https://github.com/dotnet/winforms/assets/86937911/996eabd4-94a5-482e-bcd7-4a15153a303b)
